integrated_channels/blackboard/client.py

```diff
@@ -464,3 +529,21 @@ def _get_or_create_integrated_grade_column(self, bb_course_id):
                 )
 
         return grade_column_id
+
+    def create_integration_content_for_course(self, bb_course_id, course_content_child_data):
+        """
+        Helper method to generate the default blackboard course content page with the integration custom information.
+        """
+        course_content_data = {
+            'title': 'edX Integration',
+            'position': 0,
+            "contentHandler": {"id": "resource/x-bb-folder"}
+        }
+        content_url = self.generate_create_course_content_url(bb_course_id)
+
+        content_resp = self._post(content_url, course_content_data)
+        bb_content_id = content_resp.json().get('id')
```

Comment:
This could fail with a KeyError if the id isn't present in the response.

Reply:
yes, but this is creating the content and if we don't get an ID back while also succeeding... I'd be really confused.

Reply:
However, happy to add the sanity check 👍

Reply:
Ah I think I missed the raise in the `_post` method somehow.